src/main/java/org/takes/http/BkBasic.java

```diff
@@ -108,7 +140,7 @@ private void print(final Request req, final OutputStream output)
                 )
             ).print(output);
         } finally {
-            output.close();
+            output.flush();
```

Comment:
@exper0 if you remove class sing here you should do it a some point in accept method